minor translation fixed, non-ascii error fixed

icq4ever opened the pull request:
non-ascii error fixed.
minor fixed.

arturoc commented on plugins/asciidoc/asciidoc.py:
```diff
@@ -34,6 +34,11 @@
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
+
```
the problem with this is that the asciimodule is an external plugin so if we update it this change will go away. can you try adding:

```py
## -*- coding: utf-8 -*-
```

as the first line of any file with this problem (better even in all the content files to avoid similar issues in the future)

we had this kind of issue often with blogofile and adding that header fixed it

icq4ever commented:
I removed that lines (asciidoc plugin), and added `-*- coding: utf-8 -*-` in every asciidoc. but another error occured with other files.

see log below : 
> [2016-01-26T09:26:16Z] WARNING: Nikola: Could not guess locale for language ja, using locale en_US.UTF-8
Scanning posts............done!
.  tutorials:output/ko/tutorials/index.html
########################################
TaskError - taskid:tutorials:output/ko/tutorials/index.html
PythonAction Error
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/doit/action.py", line 383, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/nikola/nikola.py", line 1143, in render_template
    template_name, None, local_context)
  File "/usr/local/lib/python2.7/dist-packages/nikola/plugins/template/mako.py", line 108, in render_template
    data = template.render_unicode(**context)
  File "/usr/lib/python2.7/dist-packages/mako/template.py", line 452, in render_unicode
    as_unicode=True)
  File "/usr/lib/python2.7/dist-packages/mako/runtime.py", line 807, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/lib/python2.7/dist-packages/mako/runtime.py", line 839, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/lib/python2.7/dist-packages/mako/runtime.py", line 865, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/icq4ever/Documents/git/ofSite/cache/.mako.tmp/base.tmpl.py", line 337, in render_body
    context['self'].content(**pageargs)
  File "/home/icq4ever/Documents/git/ofSite/cache/.mako.tmp/tutorials.mako.py", line 63, in render_content
    __M_writer(unicode(tutorials_intro))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xec in position 22: ordinal not in range(128)

as I understand, `output/ko/tutorials/index.html` is rendering from index.ko.md, a markdown file.
but log saids that might take that file as a asciidoc.

arturoc commented:
if you can remove the modifications to the asciidoc plugin i'll give it a try later

icq4ever commented:
ok I'll close this PR, and retrying
